x/ibc/applications/transfer/keeper/relay.go

```diff
@@ -98,8 +99,17 @@ func (k Keeper) SendTransfer(
 		}
 	}
 
-	labels := []metrics.Label{
-		telemetry.NewLabel("chain_id", ctx.ChainID()),
+	var labels []metrics.Label
+
+	// retrieve the counterparty chain identifier for telemetry purposes
+	_, clientState, err := k.channelKeeper.GetChannelClientState(ctx, destinationPort, destinationChannel)
```

Comment:
I forgot it'd be kind of complicated to get the counterparty chain id. I think we should just use the destPort/destChannel as the metric and the client can convert the destPort/destChannel -> chain id. Does that make sense?

Reply:
destPort/destChannel makes more sense since a single counterparty chain Id could have multiple different channels to the executing chain. This would inflate stats since not all channels will have equal importance

Reply:
I think if we use the ids then the metric becomes redundant with the send/recv packet metrics, do you think so too?

Reply:
partly yes, the source metric would be unique though

Reply:
cc @cwgoes @AdityaSripal any thoughts on this?

Reply:
let's definitely make sure not to inflate stats, destPort/destChannel seems suitable to me

Reply:
ok will add them then 👍